### DISCUSSION TITLE
Single File DB - Stalls with 2TB left of data on lvm

### DISCUSSION THREAD

**Discussion by JohnSweeney85** [paradigmxyz/reth#3369] [Technical discussions]
I have a 1TB SSD and 2TB SSD in one Volume group on a single logical volume.  Looks like it started saving the database file on the 1TB side and basically stalled at around 850GB.

Anyone have experience with this?  
Is there a better strategy to create more disk space other than buying a single 5TB disk ? 

Looks like it was allocating something to the other disk but not sure.   

![Screenshot_06-23-23_2207](https://github.com/paradigmxyz/reth/assets/5792722/6c3c142c-3523-4cf6-b3cc-9cd0e19398da)
![Screenshot_06-23-23_2207-1](https://github.com/paradigmxyz/reth/assets/5792722/73819222-331f-4365-8c38-0737737edf2c)

**Comment by 0xc926ddba** [comment]
I have a similar setup as well, with a primary 2TB (1.819 TiB) disk, and a secondary 1TB disk in my LVM volume group, so I will probably hit the threshold on my initial sync or slightly after. @JohnSweeney85 it's been 2 weeks. Do you have any ideas how to alleviate or have you already solved this? It's interesting your exporter or prometheus instance seemed to have stopped working periodically. Maybe it's because  your volume group was not set up correctly? What does your lsblk/lvs/lvdisplay output?

**Comment by onbjerg** [comment]
Just want to chime in here to acknowledge that the issue has been seen. I am not super familiar with LVM, my understanding is that the database file would be split onto both drives by the OS, i.e. that it shouldn't matter to us if you're using LVM or not as from our POV it is just a file system. Is that not the case?

**Reply by 0xc926ddba** [reply]
Yes this is exactly what should be happening with LVM, should be completely transparent to the application.

Actually I am hitting an issue with my inital sync where it seems like my sync stopped after IndexAccountHistory was completed. My disk still says 192gb free, and the symptom is that once finish becomes my header height, all my peers seem to disconnect from me and my sync gets stuck where AccountHashing sync progress < Bodies sync progress < Headers sync progress and none of them seem to increment anymore. Should I open a new ticket for this issue?

https://snapshots.raintank.io/dashboard/snapshot/C80oDckicKcftBzSE4D3NNkjE13nP6T9

**Reply by JohnSweeney85** [reply]
I am taking another stab at it right now.  I will keep yah posted.

**Comment by JohnSweeney85** [comment]
UPDATE: That was fake news. LVM works. Synced. Took quite a bit longer than expected, but works now.
